ramalama/model.py

```diff
@@ -386,7 +388,11 @@ def gpu_args(self, args, runner=False):
 
             gpu_args += [f'{args.ngl}']
 
-        if args.threads != -1:
+        # for some reason the --threads option is blowing up on Docker,
```

Comment:
This is probably something worth visiting something. It sometimes can take while for podman/docker to pull new images. I notice on podman on my machine an updated images can be posted to quay.io and podman with --pull=newer seems to not recognise there's a new "latest" image. Don't know if there's caching somewhere or what.

Even when I delete all the images and re-pull from scratch sometimes it still pulls n-1 ":latest" image.